### PR TITLE
New Web package

### PR DESCRIPTION
**Pull request by SergioCasCeb**
This pull request introduces significant changes to the current web package, including new functionality and a fresh design. The primary objective is to provide a more engaging and efficient user experience while maintaining functionality. The package is now built using Webpack, allowing to bundle all dependencies and scripts as well as minifying for production.

**Main Changes**

- New functionality and features added:
    - Multiple editors
    - New tab system for the multiple editors
    - New validation console
    - New view for the OpenAPI editor
    - New view for the AsyncAPI editor
    - New view for the Defaults editor
    - New view for the D3 and Vega visualizations
    - New examples menu
    - New settings Menu
    - Multiple themes
    
- Redesigned UI
     - New "dashboard" like design
     - New page loader
     - New validation console design
     - New save menu design
     - Light, Dark and Monochrome Themes
     - New examples menu
     - New warning pop-ups and messages
     
- Webpack Integration

### CODE REVIEW

**Comment by SergioCasCeb** [comment]
Missing featurers:

- CBOR generation
- TD/TM import from file system
- file system sync (ctrl + s) functionality
- YAML comments/anchors detection for better JSON/YAML conversion
- Show snipet of examples before utilizing it

File: packages/web-new/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Finally the web application can be deployed in production or in development mode
 
 ### Development mode
 
-1. Utilze the command `npm  run dev` which will start the webpack-dev-server in development mode, open your default browser and serve the application according to the configuaration specified in the `webpack.config.js` file.
+1. Utilze the command `npm  run dev` which will start the webpack-dev-server in development mode, generate source maps for easier debuging, open your default browser and serve the application.
```
Comment (JKRhb) [inline packages/web-new/README.md:97]: Noticed two minor typos here:

```suggestion
1. Utilize the command `npm  run dev` which will start the webpack-dev-server in development mode, generate source maps for easier debugging, open your default browser and serve the application.
```

**Comment by egekorkan** [comment]
@danielpeintner @JKRhb since you have reviewed and are committers, what do you think of the review process for such big PRs? This had to be big to get meaningful feedback and we will create follow up issues for concerns raised in the discussion threads.

**Comment by egekorkan** [comment]
Merging now to get a main branch version to work on for future updates

**Pull request merged**